### DISCUSSION TITLE
FNF 0.2.8 version 1

### DISCUSSION THREAD

**Discussion by luckydog7** [luckydog7/Funkin-android#1633] [General]
download:
https://drive.google.com/file/d/1caORgLPxYS6SyKeQfgf-ZYNxwxMuOgbS/view

<hr /><em>This discussion was created from the release <a href='https://github.com/luckydog7/Funkin-android/releases/tag/0.2.7.1.7'>FNF 0.2.8 version 1</a>.</em>

**Comment by marlnunz** [comment]
hi 7godykcul

**Reply by zsidebf** [reply]
OMG WEEK 7 YAYYYY

**Reply by theoverruleking** [reply]
It good

**Reply by GamerYTBoy** [reply]
FINALLY Week 7 is OUT!

**Comment by Sm54r** [comment]
None isn't  omittized at all please fix it

**Reply by ItzLiquid** [reply]
> GET A NEW PHONE

Shut the fuck up your annoying bitch

**Reply by Sm54r** [reply]
You the most annoying piece of shit

**Reply by SANDYTIX** [reply]
> Ты самый надоедливый кусок дерьма

and everybody give a fuck about it

**Reply by Sussysusus** [reply]
> GET A NEW PHONE

Stop being such a bitch it ain't optimized for sure the game just needs to stop crashing
Like the shaggy version

**Reply by Sunfetahu** [reply]
> > GET A NEW PHONE
> 
> Shut the fuck up your annoying bitch

Im sorry i deleted it.

**Reply by Sunfetahu** [reply]
> > GET A NEW PHONE
> 
> Stop being such a bitch it ain't optimized for sure the game just needs to stop crashing
> Like the shaggy version

I DELETED IT MAN

**Reply by khanhduy918** [reply]
Please fix blackscreen and lag

**Comment by Kappandrew** [comment]
0_0 yey

**Comment by Kappandrew** [comment]
there 's no damn chart for ugh lol

**Reply by Kappandrew** [reply]
Oh fuck its middlescroll :(

**Reply by oof9284** [reply]
theres literally no chart

**Reply by oof9284** [reply]
i disactivated it and it still cant see the notes

**Comment by Sm54r** [comment]
Why is nobody  saying that  its not  optimized

**Reply by HatsuneMiku-github** [reply]
> GET A NEW PHOEN

Shut up

**Reply by Sussysusus** [reply]
> GET A NEW PHOEN

You know how much a new phone is nigga
And also what type of phone for all they know they could spend damn days looking for an optimal phone

**Reply by Sussysusus** [reply]
> GET A NEW PHOEN

So stop saying that phones ain't free

**Comment by TrioR15** [comment]
Luckydog i want mediafire!

**Reply by EM20080** [reply]
same

**Reply by LukeTheFox** [reply]
Don't you wanna download in google drive or mega?

**Comment by DangerGamezX** [comment]
Lucky dog can u please fix it says cant download at this time):

**Reply by EM20080** [reply]
because many people downloaded it so wait 24 hours

**Reply by UnrealSticky** [reply]
you could have downloaded on mega

**Reply by Rea1408** [reply]
Mega works too, i prefer mega for a replacement of mediafire if there is no link on mediafire

**Comment by EM20080** [comment]
because many people download it so wait like 24 hours

**Comment by lilpichu** [comment]
It says it haves a error, pls mediafire

**Reply by Sunfetahu** [reply]
TRY MEGA BITCH

**Reply by HatsuneMiku-github** [reply]
> TRY MEGA BITCH

THERE IS NO MEGA

**Reply by LukeTheFox** [reply]
THERE IS MEGA

**Reply by Sunfetahu** [reply]
> > TRY MEGA BITCH
> 
> THERE IS NO MEGA

Wtf?

**Reply by Sunfetahu** [reply]
> > TRY MEGA BITCH
> 
> THERE IS NO MEGA

There is a mega

**Comment by EM20080** [comment]
well i mean ask the ppl who downloaded it

**Comment by Bloximity** [comment]
Media fire plz :^)

**Reply by Sunfetahu** [reply]
TRY MEGA

**Reply by HatsuneMiku-github** [reply]
> TRY MEGA

SHUT UP

**Reply by Sunfetahu** [reply]
> > TRY MEGA
> 
> SHUT UP

CHECK THE DOWNLOAD

**Comment by EM20080** [comment]
i cant

**Reply by jonanymus** [reply]
Do you have a release date for version 2.8.0? I'm having problems to play, I can only play for a week because if I try to play another week the game closes and it's also crashing a lot

**Comment by SavXL** [comment]
Would've been wise to have multiple download links. Google Drive stops downloads for a day whenever too many people are downloading a file. MediaFire? MEGA? Dropbox? Anything?
![Screenshot_20210709-003404_Chrome](https://user-images.githubusercontent.com/87171080/125041256-66fea480-e04d-11eb-90f6-b6e6ce52a4f5.jpg)

**Reply by ItzLiquid** [reply]
Isn't there MEGA download file?

**Reply by zsidebf** [reply]
:trollface:

**Reply by HatsuneMiku-github** [reply]
> Isn't there MEGA download file?

Nk

**Reply by HatsuneMiku-github** [reply]
*no

**Comment by EM20080** [comment]
do u have the APK pass to me mediafire link

**Reply by SavXL** [reply]
No, I don't have the APK, because I can't download it. -_-

**Reply by zsidebf** [reply]
:trollface:

**Reply by LukeTheFox** [reply]
Dude download on mega or google drive

**Reply by oof9284** [reply]
Yeah

**Comment by EM20080** [comment]
That sucks.

**Reply by lildudeguy123** [reply]
Fuck you you suck bitch go to help you asshole son of a bitch

**Reply by lildudeguy123** [reply]
Hell

**Reply by Hurbmoment** [reply]
> Fuck you you suck bitch go to help you asshole son of a bitch

BRO CHILL

**Reply by Sussysusus** [reply]
> Hell

Stop acting so childish people are allowed to have an opinion
Damn

**Comment by KnightKing777** [comment]
put on mediafire fnaf because google doesn't let you download it because too many dogs downloaded it

**Reply by UnrealSticky** [reply]
dogs xD

**Reply by LukeTheFox** [reply]
xD

**Reply by Hurbmoment** [reply]
DOGS xD

**Comment by BoyOne-gift** [comment]
LuckyDog7 throw off zip file to fix black squares please

**Reply by UnrealSticky** [reply]
xD

**Reply by LukeTheFox** [reply]
Black squares = Your phone sucks

**Comment by EM20080** [comment]
Ok, I'll check it out.

**Comment by EM20080** [comment]
you need a galaxy TAB A and also lucky dog 7 the reason why there is black
wall and glitch is because the week 7 source code isnt out yet

**Reply by fridayfunkinDav** [reply]
Hello I know this is really random but what the funk is an mediafire!?

**Reply by ItzLiquid** [reply]
If you run FNF on Android 11 it fixes black squares

**Reply by ItzLiquid** [reply]
> Hello I know this is really random but what the funk is an mediafire

The fuck? You don't what MediaFire is? It's for people who want to share their file to their friends or somethin' dumbass

**Reply by HatsuneMiku-github** [reply]
Wrong MediaFire is like Google drive

**Reply by Hurbmoment** [reply]
Black squares = GET A BETTER FUCKING PHONE

**Comment by fridayfunkinDav** [comment]
For some reason I can't play any of the songs I think is bc It's not optimized so can u pls fix it I'm so thankful for u to finally made week 7 in android btw and pls pls fix this I just really want to play the game and mod it too

**Comment by EM20080** [comment]
mediafire is a file uploading like google drive but it doesnt say "too many
downloads try again in 24 hours" no its instant

**Comment by TrioR15** [comment]
Luckydog pls optimize and lowend

**Reply by zsidebf** [reply]
The boyfriend says on my pfp: N O

**Reply by Sunfetahu** [reply]
GET A NEW PHONE

**Reply by Sm54r** [reply]
> GET A NEW p Ohio

Shut up stupid bitch

**Reply by LukeTheFox** [reply]
> Shut up stupid bitch

Dude,language.. It is the guy's opinion

**Comment by EM20080** [comment]
in the game if you want to optimized it then go to options in your game
then you go to prefrence then you go down and you see    "optimization"

**Reply by lilpichu** [reply]
The optimization doesn't works

**Reply by fridayfunkinDav** [reply]
But the problem is that when I do that I can't see anything only the background and when is about to be my turn to sing the game crashed

**Comment by GlitchyNoel** [comment]
because the game closes when I'm going to play week 7 because before the game didn't close anymore now when it comes to the cutscene the game closes

**Reply by gojira131** [reply]
Yea same

**Reply by ItzLiquid** [reply]
Works perfectly fine for me?

**Reply by elsplash** [reply]
ItzLiquid Because We Have 8 Gb Of Ram Bro

**Reply by ItzLiquid** [reply]
Oh-

**Reply by ItzLiquid** [reply]
> ItzLiquid Because We Have 8 Gb Of Ram Bro

Wait if you guys have 8 gb ram then why does it work for me? (I have 4 gb ram btw)

**Reply by LukeTheFox** [reply]
> because the game closes when I'm going to play week 7 because before the game didn't close anymore now when it comes to the cutscene the game closes

Go to options > preferences > Cutscenes. Turn it off,it works with week 6 and week 7. (Week 6= dialogue dissapears. Week 7= Tankman cutscene dissapears too)

**Comment by gojira131** [comment]
This Update removed the optimization of every week

**Comment by DangerGamezX** [comment]
Why does test not work

**Reply by zsidebf** [reply]
Enter from debug mode

**Comment by EM20080** [comment]
i dont know why well maybe it boost your game speed

**Comment by Asafte** [comment]
Need to be so much optimized

**Reply by Hurbmoment** [reply]
Ask someone to optimize the game.

**Comment by lilpichu** [comment]
It's fnf lite, that's because it crashes. Luckydog upload a bad optimized version of the game.

**Comment by lilpichu** [comment]
This stinks.

**Comment by EM20080** [comment]
That's because the source code isn't out yet

**Reply by lilpichu** [reply]
EXACTLY, THIS IS NOT REAL SOURCE CODE

**Reply by ItzLiquid** [reply]
Aye at least it has week 7

**Comment by elsplash** [comment]
omg yay but the test thingy always crashes the game even with 8 gb of ram

**Comment by Mapper66297** [comment]
> download:
> https://drive.google.com/file/d/1caORgLPxYS6SyKeQfgf-ZYNxwxMuOgbS/view
> 
> _This discussion was created from the release [FNF 0.2.8 version 1](https://github.com/luckydog7/Funkin-android/releases/tag/0.2.7.1.7)._

Finally after all these months its all here:)))

**Comment by Asafte** [comment]
A but from week 1 and week 5 crahes

lun., 12 iul. 2021, 04:26 lilpichu ***@***.***> a scris:

**Reply by lildudeguy123** [reply]
GET A NEW PHONE IDOIT

**Reply by Hurbmoment** [reply]
BITCH WHAT IF HE GETS SCAMMED

**Comment by Sm54r** [comment]
And lucky dog isn't saying anything

**Comment by GustavoDaVidaRamos13** [comment]
Anyone who wants to play, download it via mega (mirror)

**Reply by oof9284** [reply]
yes

**Comment by EM20080** [comment]
At least

**Comment by EM20080** [comment]
It does but pico will have black square ⬛⬜

**Reply by DragosDobrican** [reply]
That's probably a device problem, newer devices (like mine) do show Pico as he should, you may have an older phone which maybe a cause of outdated webGL/openGL 

**Please correct me if I'm wrong.**

**Comment by CalGamerzYTY** [comment]
Hey i want to tell you why did you need to edit the chart for other songs and the icons why can you pls put them back in the real release but you can keep the new settings and tankman also great i like it

**Reply by lildudeguy123** [reply]
Shut the fuck up

**Reply by Hurbmoment** [reply]
> Shut the fuck up

Stfu bitch

**Comment by zelaeng** [comment]
;--;

Em sáb, 24 de jul de 2021 20:37, LukeTheFox ***@***.***>
escreveu:

**Comment by ItzLiquid** [comment]
What are you bitch boy halo?

**Comment by Sunfetahu** [comment]
Sorry for swearing

**Comment by Sunfetahu** [comment]
Sorry for swearing

**Reply by iGamerProYT** [reply]
After looking at the abusive chat, I had no choice but to report you for toxic behavior

**Reply by Sunfetahu** [reply]
Did you report everybody else?

**Comment by Sunfetahu** [comment]
I barely care

**Comment by AMONGASIK** [comment]
when the full version comes out what would work Optimierung

**Comment by CalGamerzYTY** [comment]
STOPPPPPPP SWEARRRRRRRIIIIIIINNNGGNGGGGGGGG

**Reply by ChrisIsBadAtFnF** [reply]
hell

**Comment by Sunfetahu** [comment]
Ok sorry

**Comment by ItzLiquid** [comment]
Are you related to BadBoyHalo?

**Reply by Khwhw78** [reply]
LMAO ASK ANYTHING ABOUT THIS VERSION YOU IDIOT

**Comment by Sussysusus** [comment]
Can you please make the game stop crashing I usually encounter this before starting a song or during a cutscene so if you can please fix this or atleast try too

**Reply by Khwhw78** [reply]
The problem is in your phone

**Reply by oof9284** [reply]
disable the cutscenes duck.

**Reply by ChrisIsBadAtFnF** [reply]
> The problem is in your phone

Nah dude it worked fine when i turned splash notes off

**Comment by CalGamerzYTY** [comment]
No but im a kid LOL

**Reply by ghost** [reply]
get out

**Reply by Khwhw78** [reply]
Get out kid

**Reply by oof9284** [reply]
> Get out kid

yes

**Reply by CalGamerzYTY** [reply]
> Get out kid

No

**Comment by CalGamerzYTY** [comment]
I just wanted it to change the icons and the another weeks charting thats
all

**Comment by CalGamerzYTY** [comment]
Hey luckydog7 sorry but can you make the bf sprite sheet is like from the pc sprite sheet because i cannot put a mod sprite because its the pc sprite sheet. So pls make that the sprite sheet looks like the pc version

**Reply by nick8667** [reply]
If he did, the BF'll be non-optumised cause thats the thing, LuckyDog get rid of unused sprites to make more space in sprite sheet and make it work on 32-bit phones. Dunno if it somehow help, but i can give you recreates by me sprite sheet of this bf skins, that'll work with 32-bit's BF xml file. 
![B-SIDE](https://user-images.githubusercontent.com/99136774/152691957-d6396c80-74d5-47cc-98d3-0636b9416041.png)
![Fun-sized sarv](https://user-images.githubusercontent.com/99136774/152691960-962c67c1-b33a-472c-8d25-9cc6377c8b6a.png)
![NEO](https://user-images.githubusercontent.com/99136774/152691968-31ccfe0b-11a9-4f78-bb5d-d1e51ac5a8cf.png)

**Reply by CalGamerzYTY** [reply]
Srry, i was late lmao but now i know why..
Thx for some mobile sprites!

**Comment by GamerYTBoy** [comment]
Everyone who emailed all of this are sussy bakas

**Comment by GamingtasticYT** [comment]
Optimize it because my phone cant handle it and if somebody says "JUST GET A NEW PHONE DUMB@SS" im gonna f_©€|^% kill you

**Comment by Randommodder327** [comment]
Well doesn't work it just put the black box again

**Comment by LucciDarko** [comment]
E se colocasse o hitbox exatamente no lugar das setas no 'downscroll', afim de adaptar um sistema touch-screen? como o jogo piano tiles por exemplo, que clica em cima da tecla que cai

**Comment by DragosDobrican** [comment]
2 bugs I found in this game:

Test/Thorns load but just end up crashing the game
Gf's "That's how you do it!" Visuals are still present in other weeks
But there are WAY many more bugs I can find and tell you about, if I find bugs, I will report the bugs to you.
